src/DependencyInjection/MarkupNeedleExtension.php

```diff
@@ -62,6 +62,10 @@ public function load(array $configs, ContainerBuilder $container)
     private function loadDebug(array $config, ContainerBuilder $container)
     {
         $container->setParameter('markup_needle.debug', $config['debug']);
+        $container->setParameter(
```

Comment:
Minor but you could replace this with 

```
$definition = $container->findDefinition('solarium.data_collector');
$definition->addMethodCall('setSolrDashboardDomain', [$config['solr']['debug']['solr_dashboard_domain']]);
```

That way, you don't need to add service configuration calls to inject it into a service and you reduce the number of globally accessible parameters that usually are not needed in every application requrest.

Reply:
I like it. Fixed.